### ISSUE TITLE
FusionAuth requests (curl) might be blocked on certain networks 

### ISSUE THREAD

**Issue by nfebe** [PermanentOrg/devenv#64]
The local dev server would fail to reach to make `curl` connections to FusionAuth and PERHAPS other services on certain network configurations.

I experienced this on an off-grid satellite based network that is LAN-ED to a Wi-Fi router.

Not sure what configuration details about my network that would be worth sharing but `ifconfig` on the culprit network looks like this :

```
fenn@fenn-matebook:~$ ifconfig -a
br-33290810fdb9: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 172.19.0.1  netmask 255.255.0.0  broadcast 172.19.255.255
        inet6 fe80::42:26ff:fe90:e25b  prefixlen 64  scopeid 0x20<link>
        ether 02:42:26:90:e2:5b  txqueuelen 0  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 1002  bytes 197889 (197.8 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

docker0: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
        inet 172.17.0.1  netmask 255.255.0.0  broadcast 172.17.255.255
        ether 02:42:2a:82:cc:87  txqueuelen 0  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
        inet 127.0.0.1  netmask 255.0.0.0
        inet6 ::1  prefixlen 128  scopeid 0x10<host>
        loop  txqueuelen 1000  (Local Loopback)
        RX packets 27012  bytes 4408970 (4.4 MB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 27012  bytes 4408970 (4.4 MB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

vethb8ae275: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::426:baff:fe52:843a  prefixlen 64  scopeid 0x20<link>
        ether 06:26:ba:52:84:3a  txqueuelen 0  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 1118  bytes 212153 (212.1 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

wlp60s0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.1.104  netmask 255.255.255.0  broadcast 192.168.1.255
        inet6 fe80::4195:802b:6d02:7d87  prefixlen 64  scopeid 0x20<link>
        ether 38:37:8b:f1:11:47  txqueuelen 1000  (Ethernet)
        RX packets 634148  bytes 746490715 (746.4 MB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 251827  bytes 45546801 (45.5 MB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
```

**Comment by Fenn-CS** [comment]
If for some reason, everything about the environment seems to be okay but you can't login/signup try connecting to a different networks to roll out the possibility of this issue.

**Comment by slifty** [comment]
I'm closing this as not-reproducable right now; if we face it again we can open a new issue!
